server/newrelic.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+require("dotenv").config();
+
+const appName = process.env.NEW_RELIC_APP_NAME || "LogoExecutive";
```

Comment:
these environmental variables are not added into the env example file

Reply:
Added.